spec/plaindate.html

```diff
@@ -921,6 +892,35 @@ <h1>CompareISODate ( _y1_, _m1_, _d1_, _y2_, _m2_, _d2_ )</h1>
         1. If _d1_ &lt; _d2_, return -1.
         1. Return 0.
       </emu-alg>
+      </emu-clause>
+      <emu-clause id="sec-temporal-differencetemporalplaindate" aoid="DifferenceTemporalPlainDate">
+      <h1>
+        DifferenceTemporalPlainDate (
+          _direction_: 1 or -1,
+          _temporalDate_: a Temporal.PlainDate,
+          _other_: an ECMAScript language value,
+          _options_: an Object or *undefined*,
```

Comment:
```suggestion
          _options_: an ECMAScript language value,
```

Reply:
Why should we annotate options as an ECMAScript language value for some type but not the other types?